src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static final List dukeList = new List();
+    public static final String markDone = "mark";
+    public static final String bye = "bye";
+    public static final String list = "list";
+    public static final String space = " ";
+    public static final String unmarkDone = "unmark";
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        Message.printGreeting();
+        echo();
+    }
+
+    public static void echo() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
```

Comment:
maybe consider doing in.nextLine().toLowerCase() so it still works if the user accidentally makes anything uppercase